include/particle.h

```diff
@@ -215,6 +221,8 @@ class Particle : public ParticleBase<Tdim> {
   Eigen::Matrix<double, 6, Tnphases> strain_;
   //! dstrains
   Eigen::Matrix<double, 6, Tnphases> dstrain_;
+  //! strain rate
+  Eigen::VectorXd strain_rate_;  
```

Comment:
Should be specific to the phase.

Reply:
Currently, the size is 3 for 2D and 6 for 3D.
Do you want to make everything size 6 just like stress and strain?

Reply:
Yes so as to be consistent